app/views/articles/_search.html.erb

```diff
@@ -48,10 +48,10 @@
         var sortString = buildSortString(sortBy, sortDirection);
 
         if (filters) {
-          window.history.replaceState(null, null, "/search?q=" + query + "&filters=" + filters + sortString);
+          window.history.requestState(null, null, "/search?q=" + query + "&filters=" + filters + sortString);
```

Comment:
what's `requestState` ?

Reply:
@rhymes It should be requestState, my bad, apparently the commit didn't use my current save file. I'm committing the more updated file. As for using replaceState on line 51 and 54 didn't seem to make a difference between that and pushState. I left it as it is just in case it may create more unseen bugs, but for consistency I'll change it to pushState. As for the replaceState on lines 93 and 96 it seems to cause a weird bug where the user is required to click back twice for every single switch on the filters. Leaving it as replaceState doesn't produce this bug.